Sources/TripKitUI/views/results/TKUITimePickerSheet.swift

```diff
@@ -309,15 +320,43 @@ public class TKUITimePickerSheet: TKUISheet {
 
   @objc
   func doneButtonPressed(sender: Any) {
+    guard doneSelector.isEnabled 
+    else {
+      // TODO: Alert?
```

Comment:
Can you clarify this TODO? Is it safe to ignore, or when would this happen?

Reply:
this was a left over of the previous implementation that I made, updated.